src/ui/storage/index.ts

```diff
@@ -136,6 +141,27 @@ export default class ApplicationStorage implements IApplicationStorage {
     return value;
   }
 
+  async updateBindingPhraseHistory(bindingPhrase: string): Promise<void> {
```

Comment:
It would be good to generalize the concept of history for fields, so that it could be used for other items like the SSID.  Could do something similar to how show sensitive field is being stored.

Reply:
I like the idea of binding phrase history being generic text field feature. I think this feature should follow this direction.